scripts/globals/items.lua

```diff
@@ -1306,6 +1311,7 @@ xi.items =
     MEGALIXIR                       = 4254,
     PINCH_OF_MANA_POWDER            = 4255,
     SLICE_OF_DRAGON_MEAT            = 4272,
+    APPLE_AU_LAIT                   = 4300,
```

Comment:
`INSERT INTO `item_basic` VALUES (4300,0,'flask_of_apple_au_lait','apple_au_lait',1,1548,58,0,97);`
This should be `FLASK_OF_APPLE_AU_LAIT`